File: 백준/Java/Bronze/25304. 영수증/영수증.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner input = new Scanner(System.in);
        
        int total = input.nextInt();
        int num = input.nextInt();
        
        int rtotal = 0;
        
        for(int i=0; i<num; i++) {
            int price = input.nextInt();
            int item = input.nextInt();
            
            rtotal += price*item;
        }
        
        if (total == rtotal) {
            System.out.print("Yes");
        } else {
            System.out.print("No");
        }
    }
}

File: 백준/Python/Bronze/25304. 영수증/영수증.py
total = int(input())
num = int(input())

rtotal = 0

for i in range(num):
    price, item = input().split()
    price, item = int(price), int(item)
    
    rtotal += price * item
    
if rtotal == total:
    print("Yes")
else: print("No")